week-5-practise/js/main.js

```diff
@@ -0,0 +1,89 @@
+// this action will execute when a user click on the blue button and the effect will only apply to the
+//[<div> jumbotron class elements ] in dom Html
+
+var blueButton = document.querySelector("#blueBtn");
+var jumbotron = document.querySelector(".jumbotron");
+var donateAndVolunteerButtons = document.querySelectorAll(".btn-lrg");
+blueButton.addEventListener("click", donateAndVolunteersBlueButton);
+
+//this function will change the  jumbotron class background color  and the class
+//button background / style colors
+
+function donateAndVolunteersBlueButton() {
```

Comment:
Can you think of a more meaningful name for this function? Usually you'd go for a verb or an action that describes what the function does.

Reply:
@foaly-nr1  thanks for sharing the above, I find these part to be tricky sometimes, Will rename.